src/Services/ConsignmentEncode.php

```diff
@@ -120,6 +118,29 @@ public static function encodeExtraOptions(array $consignmentEncoded, AbstractCon
             ];
         }
 
+        foreach ($consignment->getMandatoryShipmentOptions() as $option) {
+            if (
+                ! isset($consignmentEncoded['options'][$option])
```

Comment:
Wat je kan doen om dit iets schoner te maken is een paar Arr functies gebruiken.

Bijvoorbeeld:

```
$key   = "options.$option";
$value = Arr::get($consignmentEncoded, $key);

if (null === $value || 0 === $value) {
    Arr::set($consignmentEncoded, $key, 1);
}
```

Nog eentje die je dan hieronder nodig gaat hebben: `Arr::forget`